addons/sourcemod/scripting/shop/item_manager.sp

```diff
@@ -2098,11 +2116,11 @@ stock void ItemManager_OnUseToggleCategory(int client, int category_id)
 		if (h_KvItems.GetNum("category_id", -1) != category_id || !h_KvItems.GetSectionName(sItemId, sizeof(sItemId)))
 			continue;
 		
-		ToggleItemCategoryOffEx(client, sItemId);
+		// ToggleItemCategoryOffEx(client, sItemId);
```

Comment:
Why this line is commented?

Reply:
> Why this line is commented?

The commented out line has been removed.

Reply:
It was added by me https://github.com/FD-Forks/Shop-Core/blame/c96d215098e4f9eb07c55cee37ccb127a717c8b5/addons/sourcemod/scripting/shop/item_manager.sp#L2101 6 years ago

Reply:
But all the same, this function is alone.

https://github.com/FD-Forks/Shop-Core/search?q=ToggleItemCategoryOffEx
![image](https://user-images.githubusercontent.com/50996383/164576503-61842163-83d1-4ea7-a625-00de8ff50719.png)

Reply:
I don't see where https://github.com/FD-Forks/Shop-Core/pull/125/files#diff-df645dc81cf2426987ac7db1517698fa108117e9c2a54dc07827c56544466a48R2107 is used. What is purpose of this function still don't understand. @R1KO do you remeber what it does?

Reply:
I think because it never get called we didn't received error from compiler.

Reply:
> I think because it never get called we didn't received error from compiler.

нет, она вызывается внутри stock функции, которая нигде не используется и комплятор её игнорирует.
![image](https://user-images.githubusercontent.com/5843406/164703362-98ff3bc2-d567-4c63-a738-cdd1b33224e8.png)

Reply:
@IL0co I think we can remove this stock function block. Because it is never used.